hoodie-utilities/src/test/resources/delta-streamer-config/dfs-source.properties

```diff
@@ -14,5 +14,14 @@
 # limitations under the License.
 #
 #
-hoodie.deltastreamer.filebased.schemaprovider.source.schema.file=file:///Users/vinoth/bin/hoodie/hoodie-utilities/src/main/resources/delta-streamer-config/source.avsc
-hoodie.deltastreamer.filebased.schemaprovider.target.schema.file=file:///Users/vinoth/bin/hoodie/hoodie-utilities/src/main/resources/delta-streamer-config/target.avsc
+#
+include=base.properties
+# Key generator props
+hoodie.datasource.write.recordkey.field=_row_key
+hoodie.datasource.write.partitionpath.field=driver
+# Schema provider props
+hoodie.deltastreamer.filebased.schemaprovider.source.schema.file=file:///Users/vinoth/bin/hoodie/hoodie-utilities/src/main/resources/delta-streamer-props/source.avsc
+hoodie.deltastreamer.filebased.schemaprovider.target.schema.file=file:///Users/vinoth/bin/hoodie/hoodie-utilities/src/main/resources/delta-streamer-props/target.avsc
```

Comment:
Should these paths be relative ?

Reply:
they don't work without being absolute.. (figuring out from cwd does not generalize well/easily to dfs. just wanted to keep this absolute to be simple).. Let me make this clear and add some dummy directory. instead.